asv_bench/benchmarks/series_methods.py

```diff
@@ -133,12 +133,24 @@ class ValueCounts:
     param_names = ["N", "dtype"]
 
     def setup(self, N, dtype):
-        self.s = Series(np.random.randint(0, N, size=10 * N)).astype(dtype)
+        self.s = Series(np.random.randint(0, N, size=10 * N)).astype("object")
```

Comment:
umm, isn't this defeating the purpose, e.g. why did this change

Reply:
Thanks, it was an unintentional change...